README.md

```diff
@@ -1,24 +1,56 @@
-# README
+# Project with two apps
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> General information
+This is a Reat Rails project.
 
-Things you may want to cover:
+## Built With
+- Rails
+- React
+- Ruby
+- postgresql
 
-* Ruby version
+# Get Started
+To get a local copy up and running follow these simple steps.
 
-* System dependencies
+```bash
```

Comment:
Please add commands `rails db:create` and `rails db:migrate` so the user is able to set up the database.